2023/02/05 10:05:03 [error] 13944#7040: *1 CreateFile() "C:\nginx-1.22.1/html/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2023/02/05 10:05:50 [notice] 13644#2944: signal process started
2023/02/05 11:03:12 [error] 13188#14976: *1 "./../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:03:17 [error] 13188#14976: *1 "./../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:03:48 [notice] 18184#6052: signal process started
2023/02/05 11:03:55 [error] 13188#14976: *2 "./../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:03:56 [error] 13188#14976: *3 "./../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:03:56 [error] 16736#18564: *4 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:03:56 [error] 16736#18564: *4 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:03:56 [error] 16736#18564: *4 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:03:56 [error] 16736#18564: *4 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:03:57 [error] 16736#18564: *4 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:03:57 [error] 16736#18564: *4 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:04:17 [notice] 10740#9652: signal process started
2023/02/05 11:04:20 [error] 16736#18564: *5 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:04:21 [error] 14384#14848: *6 "./../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:04:21 [error] 14384#14848: *6 "./../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:04:21 [error] 14384#14848: *6 "./../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:04:54 [notice] 16604#14852: signal process started
2023/02/05 11:04:56 [error] 14384#14848: *7 "./../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:04:57 [error] 19200#8396: *8 "./../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:04:57 [error] 19200#8396: *8 "./../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:04:57 [error] 19200#8396: *8 "./../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:04:57 [error] 19200#8396: *8 "./../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:04:57 [error] 19200#8396: *8 "./../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:05:16 [notice] 16252#10760: signal process started
2023/02/05 11:05:18 [error] 19200#8396: *9 "./../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:05:19 [error] 14524#4224: *10 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:05:19 [error] 14524#4224: *10 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:05:19 [error] 14524#4224: *10 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:05:19 [error] 14524#4224: *10 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:05:19 [error] 14524#4224: *10 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:05:27 [notice] 14740#960: signal process started
2023/02/05 11:05:28 [error] 14524#4224: *11 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:05:29 [error] 19080#4816: *12 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:05:29 [error] 19080#4816: *12 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:05:29 [error] 19080#4816: *12 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:05:29 [error] 19080#4816: *12 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:05:29 [error] 19080#4816: *12 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:05:30 [error] 19080#4816: *12 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:05:42 [notice] 12796#10100: signal process started
2023/02/05 11:05:43 [error] 19080#4816: *13 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:05:44 [error] 18232#12280: *14 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/root/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:05:44 [error] 18232#12280: *14 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/root/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:05:44 [error] 18232#12280: *14 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/root/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:05:44 [error] 18232#12280: *14 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/root/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:05:45 [error] 18232#12280: *14 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/root/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:01 [notice] 16008#18176: signal process started
2023/02/05 11:06:03 [error] 18232#12280: *15 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/root/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:41 [notice] 6052#18184: signal process started
2023/02/05 11:06:44 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:44 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:44 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:44 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:45 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:45 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:45 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:45 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:45 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:45 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:46 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:46 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:46 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:46 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:46 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:47 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:47 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:47 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:47 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:47 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:47 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:48 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:48 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:48 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:48 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:48 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:49 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:49 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:49 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:49 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:49 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:49 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:50 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:50 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:50 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:50 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:50 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:51 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:51 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:51 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:51 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:51 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:52 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:52 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:52 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:52 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:52 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:53 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:53 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:53 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:53 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:53 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:54 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:54 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:54 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:06:54 [error] 13240#4304: *18 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:07:01 [notice] 4100#19288: signal process started
2023/02/05 11:07:03 [error] 13240#4304: *19 "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../Desafio-14/src/public/views/layouts/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:07:17 [notice] 3784#14136: signal process started
2023/02/05 11:16:32 [notice] 6548#1832: signal process started
2023/02/05 11:16:35 [error] 13956#11568: *27 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:35 [error] 13956#11568: *27 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:35 [error] 13956#11568: *27 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:35 [error] 13956#11568: *27 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:36 [error] 13956#11568: *27 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:36 [error] 13956#11568: *27 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:36 [error] 13956#11568: *27 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:36 [error] 13956#11568: *27 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:36 [error] 13956#11568: *27 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:36 [error] 13956#11568: *27 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:37 [error] 13956#11568: *27 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:37 [error] 13956#11568: *27 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:37 [error] 13956#11568: *27 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:37 [error] 13956#11568: *27 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:37 [error] 13956#11568: *27 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:37 [error] 13956#11568: *27 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:38 [error] 13956#11568: *27 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:44 [notice] 14200#8744: signal process started
2023/02/05 11:16:46 [error] 13956#11568: *28 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:46 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:46 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:46 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:46 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:47 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:47 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:47 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:47 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:47 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:48 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:48 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:48 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:48 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:48 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:48 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:49 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:49 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:49 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:49 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:49 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:49 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:50 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:50 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:50 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:50 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:50 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:51 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:51 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:51 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:51 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:51 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:51 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:52 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:52 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:52 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:52 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:52 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:16:53 [error] 1824#8440: *29 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:17:36 [notice] 12680#4172: signal process started
2023/02/05 11:17:37 [error] 1824#8440: *30 directory index of "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/../../Desafio-14/src/public/views/" is forbidden, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost"
2023/02/05 11:19:04 [emerg] 6504#14604: unexpected "}" in F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/conf/nginx.conf:22
2023/02/05 11:19:13 [notice] 8872#3624: signal process started
2023/02/05 11:53:51 [error] 404#15224: *38 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/auth/register HTTP/1.1", upstream: "http://[::1]:8080/api/auth/register", host: "localhost", referrer: "http://localhost/api/auth/login"
2023/02/05 11:53:54 [error] 404#15224: *39 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/auth/login HTTP/1.1", upstream: "http://127.0.0.1:8080/api/auth/login", host: "localhost"
2023/02/05 11:53:54 [error] 404#15224: *39 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/auth/login HTTP/1.1", upstream: "http://node_app/api/auth/login", host: "localhost"
2023/02/05 11:53:59 [notice] 17088#308: signal process started
2023/02/05 11:54:01 [error] 404#15224: *41 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/auth/login HTTP/1.1", upstream: "http://node_app/api/auth/login", host: "localhost"
2023/02/05 11:54:04 [error] 2144#3612: *44 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/auth/login HTTP/1.1", upstream: "http://[::1]:8080/api/auth/login", host: "localhost"
2023/02/05 11:54:06 [error] 2144#3612: *44 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/auth/login HTTP/1.1", upstream: "http://127.0.0.1:8080/api/auth/login", host: "localhost"
2023/02/05 11:54:18 [notice] 12968#15936: signal process started
2023/02/05 11:54:22 [error] 2144#3612: *46 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/auth/login HTTP/1.1", upstream: "http://127.0.0.1:8080/api/auth/login", host: "localhost"
2023/02/05 11:54:24 [error] 2144#3612: *46 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/auth/login HTTP/1.1", upstream: "http://[::1]:8080/api/auth/login", host: "localhost"
2023/02/05 16:33:12 [error] 12100#18692: *82 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /info HTTP/1.1", upstream: "http://[::1]:8080/info", host: "localhost"
2023/02/05 16:33:14 [error] 12100#18692: *82 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /info HTTP/1.1", upstream: "http://127.0.0.1:8080/info", host: "localhost"
2023/02/05 16:40:26 [error] 12100#18692: *88 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /info HTTP/1.1", upstream: "http://127.0.0.1:8080/info", host: "localhost"
2023/02/05 16:40:27 [notice] 11456#9772: signal process started
2023/02/05 16:40:28 [error] 12100#18692: *88 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /info HTTP/1.1", upstream: "http://[::1]:8080/info", host: "localhost"
2023/02/05 16:40:30 [error] 12100#18692: *90 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /info HTTP/1.1", upstream: "http://node_app/info", host: "localhost"
2023/02/05 16:40:36 [error] 11792#8836: *93 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:8080/", host: "localhost"
2023/02/05 16:40:38 [error] 11792#8836: *93 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8080/", host: "localhost"
2023/02/05 17:52:17 [notice] 14576#18856: signal process started
2023/02/05 18:05:07 [notice] 15368#13288: signal process started
2023/02/05 18:41:40 [notice] 5724#3812: signal process started
2023/02/05 18:46:33 [notice] 2984#3848: signal process started
2023/02/05 19:15:53 [notice] 9660#2256: signal process started
2023/02/05 19:25:34 [emerg] 13212#2260: unexpected "}" in F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/conf/nginx.conf:34
2023/02/05 19:25:43 [notice] 13876#6360: signal process started
2023/02/05 19:25:55 [error] 3352#5876: *97 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:8080/", host: "localhost"
2023/02/05 19:25:58 [error] 3352#5876: *99 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8080/", host: "localhost"
2023/02/05 19:25:58 [error] 3352#5876: *99 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/05 19:26:07 [error] 3352#5876: *99 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://[::1]:8081/api/randoms", host: "localhost"
2023/02/05 19:26:27 [error] 3352#5876: *103 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://127.0.0.1:8081/api/randoms", host: "localhost"
2023/02/05 19:26:29 [error] 3352#5876: *103 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://[::1]:8081/api/randoms", host: "localhost"
2023/02/05 19:26:31 [error] 3352#5876: *103 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app2/api/randoms", host: "localhost"
2023/02/05 19:27:02 [error] 3352#5876: *103 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://127.0.0.1:8081/api/randoms", host: "localhost"
2023/02/05 19:27:04 [error] 3352#5876: *103 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://[::1]:8081/api/randoms", host: "localhost"
2023/02/05 19:37:48 [notice] 7520#15340: signal process started
2023/02/05 19:38:05 [notice] 8576#2712: signal process started
2023/02/05 19:38:25 [notice] 16520#14696: signal process started
2023/02/05 19:51:53 [notice] 16760#2760: signal process started
2023/02/05 20:11:15 [notice] 15368#10740: signal process started
2023/02/05 20:13:36 [notice] 14288#9332: signal process started
2023/02/05 20:15:40 [notice] 3164#5048: signal process started
2023/02/05 20:18:05 [notice] 2096#19072: signal process started
2023/02/05 20:18:24 [error] 15888#6896: *3 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://[::1]:8081/api/randoms", host: "localhost"
2023/02/05 20:18:26 [error] 15888#6896: *3 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://127.0.0.1:8081/api/randoms", host: "localhost"
2023/02/05 20:18:29 [error] 15888#6896: *3 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app2/api/randoms", host: "localhost"
2023/02/05 20:18:58 [error] 15888#6896: *3 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://127.0.0.1:8081/api/randoms", host: "localhost"
2023/02/05 20:19:00 [error] 15888#6896: *3 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://[::1]:8081/api/randoms", host: "localhost"
2023/02/05 20:19:36 [notice] 3164#13680: signal process started
2023/02/05 20:22:36 [error] 12036#18500: *11 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://[::1]:8081/api/randoms", host: "localhost"
2023/02/05 20:22:38 [error] 12036#18500: *11 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://127.0.0.1:8081/api/randoms", host: "localhost"
2023/02/05 20:22:42 [error] 12036#18500: *11 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app2/api/randoms", host: "localhost"
2023/02/05 20:23:15 [notice] 20468#19928: signal process started
2023/02/05 20:23:19 [error] 12036#18500: *12 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://127.0.0.1:8081/api/randoms", host: "localhost"
2023/02/05 20:23:21 [error] 12036#18500: *12 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://[::1]:8081/api/randoms", host: "localhost"
2023/02/05 20:23:23 [error] 12036#18500: *14 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app2/api/randoms", host: "localhost"
2023/02/05 20:23:31 [error] 13456#18112: *32 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://127.0.0.1:8081/api/randoms", host: "localhost"
2023/02/05 20:23:33 [error] 13456#18112: *32 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://[::1]:8081/api/randoms", host: "localhost"
2023/02/05 20:25:20 [notice] 16504#9816: signal process started
2023/02/05 20:31:38 [notice] 20256#19876: signal process started
2023/02/05 20:41:40 [notice] 6748#20272: signal process started
2023/02/05 20:46:51 [notice] 4092#7312: signal process started
2023/02/05 20:46:52 [notice] 15308#9360: signal process started
2023/02/05 20:47:02 [error] 7960#7276: *36 CreateFile() "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/html/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "localhost:8081", referrer: "http://localhost:8081/"
2023/02/05 20:47:13 [error] 7960#7276: *36 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://[::1]:8081/api/randoms", host: "localhost:8081"
2023/02/05 20:47:13 [alert] 7960#7276: 512 worker_connections are not enough
2023/02/05 20:47:13 [error] 7960#7276: *547 WSARecv() failed (10053: An established connection was aborted by the software in your host machine) while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.0", upstream: "http://127.0.0.1:8081/api/randoms", host: "node_app2"
2023/02/05 20:47:13 [error] 7960#7276: *547 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.0", upstream: "http://node_app2/api/randoms", host: "node_app2"
2023/02/05 20:48:43 [notice] 3460#19320: signal process started
2023/02/05 20:48:47 [notice] 12180#10916: signal process started
2023/02/06 20:13:59 [notice] 11704#2532: signal process started
2023/02/06 20:15:03 [error] 7684#12804: *1 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/auth/login HTTP/1.1", upstream: "http://[::1]:8080/api/auth/login", host: "localhost"
2023/02/06 20:15:05 [error] 7684#12804: *1 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/auth/login HTTP/1.1", upstream: "http://127.0.0.1:8080/api/auth/login", host: "localhost"
2023/02/06 20:15:07 [error] 7684#12804: *1 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/auth/login HTTP/1.1", upstream: "http://node_app/api/auth/login", host: "localhost"
2023/02/06 20:15:07 [error] 7684#12804: *1 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/auth/login HTTP/1.1", upstream: "http://node_app/api/auth/login", host: "localhost"
2023/02/06 20:15:07 [error] 7684#12804: *1 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/auth/login HTTP/1.1", upstream: "http://node_app/api/auth/login", host: "localhost"
2023/02/06 20:15:07 [error] 7684#12804: *1 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/auth/login HTTP/1.1", upstream: "http://node_app/api/auth/login", host: "localhost"
2023/02/06 20:15:07 [error] 7684#12804: *1 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/auth/login HTTP/1.1", upstream: "http://node_app/api/auth/login", host: "localhost"
2023/02/06 20:15:08 [error] 7684#12804: *1 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/auth/login HTTP/1.1", upstream: "http://node_app/api/auth/login", host: "localhost"
2023/02/06 20:15:08 [error] 7684#12804: *1 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/auth/login HTTP/1.1", upstream: "http://node_app/api/auth/login", host: "localhost"
2023/02/06 20:15:08 [error] 7684#12804: *1 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/auth/login HTTP/1.1", upstream: "http://node_app/api/auth/login", host: "localhost"
2023/02/06 20:15:08 [error] 7684#12804: *1 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/auth/login HTTP/1.1", upstream: "http://node_app/api/auth/login", host: "localhost"
2023/02/06 20:17:40 [notice] 12332#19048: signal process started
2023/02/06 20:23:37 [error] 7684#12804: *15 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8080/", host: "127.0.0.1"
2023/02/06 20:23:39 [error] 7684#12804: *15 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:8080/", host: "127.0.0.1"
2023/02/06 20:23:39 [error] 7684#12804: *15 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://node_app/favicon.ico", host: "127.0.0.1", referrer: "http://127.0.0.1/"
2023/02/06 20:24:05 [notice] 14116#7164: signal process started
2023/02/06 20:24:08 [error] 7684#12804: *15 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8080/", host: "127.0.0.1"
2023/02/06 20:24:10 [error] 7684#12804: *15 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:8080/", host: "127.0.0.1"
2023/02/06 20:24:10 [error] 7684#12804: *15 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://node_app/favicon.ico", host: "127.0.0.1", referrer: "http://127.0.0.1/"
2023/02/06 20:24:45 [notice] 9588#6268: signal process started
2023/02/06 20:24:58 [notice] 5032#256: signal process started
2023/02/06 20:25:11 [error] 7684#12804: *21 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8080/", host: "localhost"
2023/02/06 20:25:13 [error] 7684#12804: *21 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:8080/", host: "localhost"
2023/02/06 20:25:16 [error] 7684#12804: *25 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "127.0.0.1"
2023/02/06 20:25:16 [error] 7684#12804: *25 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://node_app/favicon.ico", host: "127.0.0.1", referrer: "http://127.0.0.1/"
2023/02/06 20:25:20 [error] 7684#12804: *25 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "127.0.0.1"
2023/02/06 20:25:20 [error] 7684#12804: *25 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://node_app/favicon.ico", host: "127.0.0.1", referrer: "http://127.0.0.1/"
2023/02/06 20:25:21 [error] 7684#12804: *25 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "127.0.0.1"
2023/02/06 20:25:21 [error] 7684#12804: *25 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://node_app/favicon.ico", host: "127.0.0.1", referrer: "http://127.0.0.1/"
2023/02/06 20:25:21 [error] 7684#12804: *25 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "127.0.0.1"
2023/02/06 20:25:21 [error] 7684#12804: *25 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://node_app/favicon.ico", host: "127.0.0.1", referrer: "http://127.0.0.1/"
2023/02/06 20:25:21 [error] 7684#12804: *25 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "127.0.0.1"
2023/02/06 20:25:21 [error] 7684#12804: *25 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://node_app/favicon.ico", host: "127.0.0.1", referrer: "http://127.0.0.1/"
2023/02/06 20:25:24 [error] 7684#12804: *25 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8080/", host: "127.0.0.1"
2023/02/06 20:25:26 [error] 7684#12804: *25 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:8080/", host: "127.0.0.1"
2023/02/06 20:25:26 [error] 7684#12804: *25 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://node_app/favicon.ico", host: "127.0.0.1", referrer: "http://127.0.0.1/"
2023/02/06 20:25:37 [error] 7684#12804: *21 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8080/", host: "localhost"
2023/02/06 20:25:39 [error] 7684#12804: *21 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:8080/", host: "localhost"
2023/02/06 20:25:40 [notice] 8696#7400: signal process started
2023/02/06 20:25:42 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:25:43 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:25:43 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:25:43 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:25:43 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:25:43 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:25:44 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:25:44 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:25:44 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:25:44 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:25:44 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:25:45 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:25:45 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:25:45 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:25:46 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:25:52 [error] 7684#12804: *21 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8080/", host: "localhost"
2023/02/06 20:25:54 [error] 7684#12804: *21 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:8080/", host: "localhost"
2023/02/06 20:25:55 [notice] 18620#8512: signal process started
2023/02/06 20:26:00 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:26:00 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:26:00 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:26:01 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:26:01 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:26:01 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:26:01 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:26:01 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:26:02 [error] 7684#12804: *21 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:26:05 [error] 7684#12804: *21 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8080/", host: "localhost"
2023/02/06 20:26:07 [error] 7684#12804: *21 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:8080/", host: "localhost"
2023/02/06 20:26:43 [error] 7684#12804: *21 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8080/", host: "localhost"
2023/02/06 20:26:45 [error] 7684#12804: *21 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:8080/", host: "localhost"
2023/02/06 20:26:52 [notice] 6268#13196: signal process started
2023/02/06 20:27:01 [error] 7684#12804: *36 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8080/", host: "localhost"
2023/02/06 20:27:03 [error] 7684#12804: *36 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:8080/", host: "localhost"
2023/02/06 20:27:05 [error] 7684#12804: *36 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:27:07 [error] 7684#12804: *36 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:27:08 [error] 7684#12804: *36 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:27:08 [error] 7684#12804: *36 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:27:08 [error] 7684#12804: *36 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:27:08 [error] 7684#12804: *36 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/06 20:27:39 [notice] 18264#8216: signal process started
2023/02/06 20:29:08 [error] 12308#3844: *1 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:8080/", host: "localhost"
2023/02/06 20:29:10 [error] 12308#3844: *1 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8080/", host: "localhost"
2023/02/06 20:29:18 [notice] 6344#16924: signal process started
2023/02/06 20:29:22 [error] 12308#3844: *4 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:8080/", host: "localhost"
2023/02/06 20:29:24 [error] 12308#3844: *4 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8080/", host: "localhost"
2023/02/06 20:40:07 [notice] 1608#4808: signal process started
2023/02/06 20:40:17 [error] 12768#16912: *17 CreateFile() "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/html/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "localhost:8081", referrer: "http://localhost:8081/"
2023/02/06 20:40:32 [error] 12768#16912: *17 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://[::1]:8081/api/randoms", host: "localhost:8081"
2023/02/06 20:40:32 [alert] 12768#16912: 512 worker_connections are not enough
2023/02/06 20:40:32 [error] 12768#16912: *528 WSARecv() failed (10053: An established connection was aborted by the software in your host machine) while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.0", upstream: "http://127.0.0.1:8081/api/randoms", host: "node_app2"
2023/02/06 20:40:32 [error] 12768#16912: *528 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.0", upstream: "http://node_app2/api/randoms", host: "node_app2"
2023/02/06 20:40:52 [notice] 404#14628: signal process started
2023/02/06 20:41:11 [notice] 15336#19748: signal process started
2023/02/06 20:41:19 [error] 3424#20136: *536 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://[::1]:8081/api/randoms", host: "localhost:8081"
2023/02/06 20:41:19 [alert] 3424#20136: 512 worker_connections are not enough
2023/02/06 20:41:19 [error] 3424#20136: *1046 WSARecv() failed (10053: An established connection was aborted by the software in your host machine) while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.0", upstream: "http://127.0.0.1:8081/api/randoms", host: "node_app2"
2023/02/06 20:41:19 [error] 3424#20136: *1046 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.0", upstream: "http://node_app2/api/randoms", host: "node_app2"
2023/02/06 20:41:22 [error] 3424#20136: *536 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app2/api/randoms", host: "localhost:8081"
2023/02/06 20:41:23 [error] 3424#20136: *536 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app2/api/randoms", host: "localhost:8081"
2023/02/06 20:41:24 [error] 3424#20136: *536 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app2/api/randoms", host: "localhost:8081"
2023/02/06 20:41:24 [error] 3424#20136: *536 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app2/api/randoms", host: "localhost:8081"
2023/02/06 20:41:24 [error] 3424#20136: *536 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app2/api/randoms", host: "localhost:8081"
2023/02/06 20:41:24 [error] 3424#20136: *536 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app2/api/randoms", host: "localhost:8081"
2023/02/06 20:41:27 [error] 3424#20136: *536 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app2/api/randoms", host: "localhost:8081"
2023/02/06 20:41:29 [notice] 16296#14444: signal process started
2023/02/06 20:41:33 [error] 12916#10484: *1050 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.0", upstream: "http://[::1]:8081/api/randoms", host: "node_app2"
2023/02/06 20:41:33 [alert] 12916#10484: 512 worker_connections are not enough
2023/02/06 20:41:33 [error] 12916#10484: *1560 WSARecv() failed (10053: An established connection was aborted by the software in your host machine) while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.0", upstream: "http://127.0.0.1:8081/api/randoms", host: "node_app2"
2023/02/06 20:41:33 [error] 12916#10484: *1560 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.0", upstream: "http://node_app2/api/randoms", host: "node_app2"
2023/02/06 20:41:34 [error] 12916#10484: *1562 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app2/api/randoms", host: "localhost:8081"
2023/02/06 20:41:36 [error] 12916#10484: *1562 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app2/api/randoms", host: "localhost:8081"
2023/02/06 22:50:37 [notice] 8380#14812: signal process started
2023/02/06 22:51:29 [error] 13664#18156: *1565 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://[::1]:8081/api/randoms", host: "localhost"
2023/02/06 22:51:31 [error] 13664#18156: *1565 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://127.0.0.1:8081/api/randoms", host: "localhost"
2023/02/06 22:51:31 [error] 13664#18156: *1566 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app_random/api/randoms", host: "localhost"
2023/02/06 22:51:33 [error] 13664#18156: *1566 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app_random/api/randoms", host: "localhost"
2023/02/06 22:51:37 [notice] 6380#5764: signal process started
2023/02/06 22:51:39 [error] 13664#18156: *1569 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app_random/api/randoms", host: "localhost"
2023/02/06 22:51:42 [error] 16056#4948: *1574 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://127.0.0.1:8081/api/randoms", host: "localhost"
2023/02/06 22:51:44 [error] 16056#4948: *1574 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://[::1]:8081/api/randoms", host: "localhost"
2023/02/06 22:53:17 [notice] 13428#15284: signal process started
2023/02/06 22:54:22 [warn] 17460#15464: conflicting server name "localhost" on 0.0.0.0:80, ignored
2023/02/06 22:54:22 [notice] 17460#15464: signal process started
2023/02/06 22:54:22 [warn] 11972#13440: conflicting server name "localhost" on 0.0.0.0:80, ignored
2023/02/06 22:54:22 [warn] 3152#18752: conflicting server name "localhost" on 0.0.0.0:80, ignored
2023/02/06 22:54:22 [warn] 6624#16304: conflicting server name "localhost" on 0.0.0.0:80, ignored
2023/02/06 22:54:22 [warn] 18828#364: conflicting server name "localhost" on 0.0.0.0:80, ignored
2023/02/06 22:54:22 [warn] 13748#12632: conflicting server name "localhost" on 0.0.0.0:80, ignored
2023/02/06 22:54:22 [warn] 19708#3404: conflicting server name "localhost" on 0.0.0.0:80, ignored
2023/02/06 22:54:22 [warn] 872#10600: conflicting server name "localhost" on 0.0.0.0:80, ignored
2023/02/06 22:54:22 [warn] 2392#14700: conflicting server name "localhost" on 0.0.0.0:80, ignored
2023/02/06 22:54:22 [warn] 8444#12560: conflicting server name "localhost" on 0.0.0.0:80, ignored
2023/02/06 22:54:22 [warn] 8512#17980: conflicting server name "localhost" on 0.0.0.0:80, ignored
2023/02/06 22:54:22 [warn] 13328#10860: conflicting server name "localhost" on 0.0.0.0:80, ignored
2023/02/06 22:54:22 [warn] 13092#8928: conflicting server name "localhost" on 0.0.0.0:80, ignored
2023/02/06 22:54:22 [warn] 19388#6324: conflicting server name "localhost" on 0.0.0.0:80, ignored
2023/02/06 22:54:22 [warn] 11164#17944: conflicting server name "localhost" on 0.0.0.0:80, ignored
2023/02/06 22:54:22 [warn] 17664#19480: conflicting server name "localhost" on 0.0.0.0:80, ignored
2023/02/06 22:54:22 [warn] 17628#5672: conflicting server name "localhost" on 0.0.0.0:80, ignored
2023/02/06 22:54:38 [notice] 19860#10640: signal process started
2023/02/06 22:54:48 [error] 11764#15472: *1578 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://[::1]:8081/api/randoms", host: "localhost"
2023/02/06 22:54:50 [error] 11764#15472: *1578 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://127.0.0.1:8081/api/randoms", host: "localhost"
2023/02/06 22:54:52 [error] 11764#15472: *1578 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app_random/api/randoms", host: "localhost"
2023/02/06 22:54:54 [notice] 11892#2716: signal process started
2023/02/06 22:54:56 [error] 11764#15472: *1579 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app_random/api/randoms", host: "localhost"
2023/02/06 22:55:20 [notice] 5940#14440: signal process started
2023/02/06 23:14:21 [notice] 8392#5500: signal process started
2023/02/07 16:52:05 [error] 4284#4112: *1 CreateFile() "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/html/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2023/02/07 16:52:05 [error] 4284#4112: *2 CreateFile() "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/html/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2023/02/07 17:04:16 [notice] 20288#404: signal process started
2023/02/07 17:08:17 [notice] 2012#2120: signal process started
2023/02/07 17:08:28 [error] 18928#12452: *11 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /info HTTP/1.1", upstream: "http://[::1]:8080/info", host: "localhost"
2023/02/07 17:08:30 [error] 18928#12452: *11 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /info HTTP/1.1", upstream: "http://127.0.0.1:8080/info", host: "localhost"
2023/02/07 17:08:32 [error] 18928#12452: *11 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /info HTTP/1.1", upstream: "http://node_app/info", host: "localhost"
2023/02/07 17:08:38 [error] 18928#12452: *11 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /info HTTP/1.1", upstream: "http://node_app/info", host: "localhost"
2023/02/07 17:09:24 [notice] 12104#13844: signal process started
2023/02/07 17:09:27 [notice] 17732#10640: signal process started
2023/02/07 17:09:31 [error] 18580#18228: *15 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:8080/", host: "localhost"
2023/02/07 17:09:33 [error] 18580#18228: *15 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8080/", host: "localhost"
2023/02/07 17:09:35 [error] 18580#18228: *15 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://node_app/", host: "localhost"
2023/02/07 17:11:40 [emerg] 12808#11356: invalid number of arguments in "root" directive in F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\nginx/conf/nginx.conf:28
2023/02/07 17:11:59 [notice] 13784#18880: signal process started
2023/02/07 17:24:13 [notice] 340#4800: signal process started
2023/02/07 17:39:16 [notice] 20332#5052: signal process started
2023/02/07 17:42:30 [error] 18392#13388: *31 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://[::1]:8081/api/randoms", host: "localhost"
2023/02/07 17:42:34 [error] 18392#13388: *33 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://127.0.0.1:8081/api/randoms", host: "localhost"
2023/02/07 17:42:34 [error] 18392#13388: *33 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app_random/api/randoms", host: "localhost"
2023/02/07 17:42:38 [notice] 9400#16168: signal process started
2023/02/07 17:42:42 [error] 8104#1308: *36 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://[::1]:8081/api/randoms", host: "localhost"
2023/02/07 17:42:44 [error] 8104#1308: *36 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://127.0.0.1:8081/api/randoms", host: "localhost"
2023/02/07 17:42:45 [error] 8104#1308: *36 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app_random/api/randoms", host: "localhost"
2023/02/07 17:42:46 [error] 8104#1308: *36 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app_random/api/randoms", host: "localhost"
2023/02/07 17:42:47 [error] 8104#1308: *36 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app_random/api/randoms", host: "localhost"
2023/02/07 17:42:47 [error] 8104#1308: *36 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app_random/api/randoms", host: "localhost"
2023/02/07 17:42:47 [error] 8104#1308: *36 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app_random/api/randoms", host: "localhost"
2023/02/07 17:42:48 [error] 8104#1308: *36 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app_random/api/randoms", host: "localhost"
2023/02/07 17:42:48 [error] 8104#1308: *36 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app_random/api/randoms", host: "localhost"
2023/02/07 17:42:50 [error] 8104#1308: *36 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app_random/api/randoms", host: "localhost"
2023/02/07 17:45:16 [notice] 20244#13552: signal process started
2023/02/07 17:45:32 [error] 8104#1308: *41 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://127.0.0.1:8081/api/randoms", host: "localhost"
2023/02/07 17:45:34 [error] 8104#1308: *41 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://[::1]:8081/api/randoms", host: "localhost"
2023/02/07 17:46:21 [notice] 4584#5012: signal process started
2023/02/07 17:46:38 [error] 18068#3764: *45 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://[::1]:8081/api/randoms", host: "localhost"
2023/02/07 17:46:40 [error] 18068#3764: *45 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://127.0.0.1:8081/api/randoms", host: "localhost"
2023/02/07 17:46:41 [error] 18068#3764: *45 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app_random/api/randoms", host: "localhost"
2023/02/07 17:46:41 [error] 18068#3764: *45 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app_random/api/randoms", host: "localhost"
2023/02/07 17:46:42 [error] 18068#3764: *45 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app_random/api/randoms", host: "localhost"
2023/02/07 17:57:11 [notice] 15368#3424: signal process started
2023/02/07 18:04:38 [notice] 9100#12576: signal process started
2023/02/07 18:04:51 [error] 9940#18844: *56 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://[::1]:8081/api/randoms", host: "localhost"
2023/02/07 18:04:53 [error] 9940#18844: *56 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://127.0.0.1:8081/api/randoms", host: "localhost"
2023/02/07 18:04:55 [error] 9940#18844: *56 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app_random/api/randoms", host: "localhost"
2023/02/07 18:05:25 [error] 9940#18844: *57 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://127.0.0.1:8081/api/randoms", host: "localhost"
2023/02/07 18:05:27 [error] 9940#18844: *57 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://[::1]:8081/api/randoms", host: "localhost"
2023/02/07 18:54:04 [notice] 3220#17496: signal process started
2023/02/09 17:59:07 [notice] 16404#8900: signal process started
2023/02/09 17:59:48 [error] 8536#4104: *1 CreateFile() "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\src\public/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2023/02/09 17:59:53 [error] 8536#4104: *1 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://[::1]:8081/api/randoms", host: "localhost"
2023/02/09 17:59:55 [error] 8536#4104: *1 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://127.0.0.1:8081/api/randoms", host: "localhost"
2023/02/09 17:59:59 [error] 8536#4104: *1 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://node_app_random/api/randoms", host: "localhost"
2023/02/09 18:01:11 [notice] 20384#4856: signal process started
2023/02/09 18:12:49 [notice] 6012#19128: signal process started
2023/02/10 20:10:30 [error] 15388#15896: *1 CreateFile() "F:\CURSO CODERHOUSE\BACKEND\---------DESAFIOS-------------\Desafios\Desafio-14\src\public/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2023/02/10 20:14:35 [error] 15388#15896: *8 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://[::1]:8081/api/randoms", host: "localhost"
2023/02/10 20:14:37 [error] 15388#15896: *8 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/randoms HTTP/1.1", upstream: "http://127.0.0.1:8081/api/randoms", host: "localhost"
2023/02/10 20:47:28 [notice] 5384#14664: signal process started
